usr/share/rear/setup/default/010_pre_recovery_script.sh

```diff
@@ -1,4 +1,11 @@
 if test "$PRE_RECOVERY_SCRIPT" ; then
-	Log "Running PRE_RECOVERY_SCRIPT '${PRE_RECOVERY_SCRIPT[@]}'"
-	eval "${PRE_RECOVERY_SCRIPT[@]}"
+    Log "Running PRE_RECOVERY_SCRIPT '${PRE_RECOVERY_SCRIPT[@]}'"
+    eval "${PRE_RECOVERY_SCRIPT[@]}"
 fi
+
+for command in "${PRE_RECOVERY_COMMANDS[@]}"; do
+    Log "Running PRE_RECOVERY_COMMANDS: '$command'"
+    eval "$command"
+done
+
+# vim: set et ts=4 sw=4:
```

Comment:
ditto, this can be done together with formatting changes in the same commit, separate from code changes